app/assets/javascripts/registration/grid.js.erb

```diff
@@ -470,6 +470,17 @@ var gridContext = function() {
           }
         })
 
+        // Show/hide external workflow drop-down when workflow selector changes
```

Comment:
@cbeer and @peetucket, i think i might have something that addresses basic UI/integration testing.  it's not particularly comprehensive considering that the page is a big dynamic JS edit form, but it at least tests the basic use cases for what peter added and for item registration in general.  thanks to @mejackreed for the help in debugging.

see https://github.com/sul-dlss/argo/pull/946